pkg/validation/validators/validators_test.go

```diff
@@ -19,33 +19,33 @@ func TestFinalNewline(t *testing.T) {
 		{"x\r", "true", "cr", nil},
 		{"x\r\n", "true", "crlf", nil},
 
-		{"x", "true", "lf", errors.New("Wrong line endings or no final newline")},
-		{"x", "true", "cr", errors.New("Wrong line endings or no final newline")},
-		{"x", "true", "crlf", errors.New("Wrong line endings or no final newline")},
+		{"x", "true", "lf", errors.New("Final newline expected")},
+		{"x", "true", "cr", errors.New("Final newline expected")},
+		{"x", "true", "crlf", errors.New("Final newline expected")},
 
-		{"x\n", "true", "cr", errors.New("Wrong line endings or no final newline")},
-		{"x\n", "true", "crlf", errors.New("Wrong line endings or no final newline")},
+		{"x\n", "true", "cr", errors.New("Final newline expected")},
+		{"x\n", "true", "crlf", errors.New("Final newline expected")},
 
-		{"x\r", "true", "lf", errors.New("Wrong line endings or no final newline")},
-		{"x\r", "true", "crlf", errors.New("Wrong line endings or no final newline")},
+		{"x\r", "true", "lf", errors.New("Final newline expected")},
+		{"x\r", "true", "crlf", errors.New("Final newline expected")},
```

Comment:
The error messages here, if I understand correctly, should be `"Wrong line endings"`.

For example line `26`, with `"x\n"` with `cr` (`\r`) mode, it is because the final line ending should be `\r` and it is instead `\n`, but there is a final newline, it is just not the right one?

Reply:
That's what I meant in the commit message: There's validators.LineEnding, that catches this error already:

```console
$ printf 'x\r' > test
$ ./bin/ec -format gcc test
test:
        Final newline expected
        Not all lines have the correct end of line character

2 errors found
```

I found it more logical to not test it twice, but expect `\r\n` as final newline and `\r\n` is not here.